### DISCUSSION TITLE
Welcome to Cyberbear.github.io Discussions!

### DISCUSSION THREAD

**Discussion by Vinay-Khanagavi** [Vinay-Khanagavi/Cyberbear.github.io#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Crimson-Typhoon-147** [comment]
hey @Vinay-Khanagavi I proposed https://github.com/Vinay-Khanagavi/Cyberbear.github.io/issues/101 this Feature 2 days before  but I also had raised a PR on https://github.com/Vinay-Khanagavi/Cyberbear.github.io/issues/95 and the maintainer had asked to delete the file which created some confusion that if I either have to delete the whole file or the chnages that I had made but eventually the PR got closed and i was told to work on the issue 101 once my previous PR get merged . So here I need your confirmation if I should start working on the issue 101 or not?
